var/sabath/assets/sabath/models/c/cosmoflow/cosmoflow.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/configs/cosmo.yaml b/configs/cosmo.yaml
+index 675e32f..d164966 100644
+--- a/configs/cosmo.yaml
++++ b/configs/cosmo.yaml
+@@ -8,10 +8,10 @@ mlperf:
+ 
+ data:
+     name: cosmo
+-    data_dir: /global/cscratch1/sd/sfarrell/cosmoflow-benchmark/data/cosmoUniverse_2019_05_4parE_tf_v2
++    data_dir: /path/to/cosmoUniverse_2019_05_4parE_tf_v2_mini/ #Update path according where your dataset is located
+     compression: GZIP
+-    n_train: 524288
+-    n_valid: 65536
++    n_train: 1024
```

Comment:
Where do these numbers come from? How do they change between data sets? Why they need to be changed at all?

Reply:
These numbers, (n_train and n_valid) represent the number of files within the train and validation directories respectively of the dataset.
the previous numbers (n_train: 524288 and n_valid: 65536) are for the larger dataset. And the changed numbers (n_train: 1024 and n_valid: 128) are for the mini dataset.

Reply:
This has to become part of SABATH that figures out the parameters of the data set and sets these values for different comsoflow data sets that we have or even more that we'll have in the future.